src/Frontend/Core/Layout/Templates/Breadcrumb.html.twig

```diff
@@ -1,7 +1,7 @@
 <span class="sr-only">{{ 'lbl.YouAreHere'|trans }}</span>
 <ul class="breadcrumb" itemscope itemtype="https://schema.org/BreadcrumbList" role="navigation">
   {% for breadcrumb in breadcrumb %}
-    <li{% if loop.last %} class="active"{% endif %} itemprop="itemListElement" itemscope
+    <li{% if loop.last %} class="breadcrumb-item{% endif %}" itemprop="itemListElement" itemscope
```

Comment:
it always needs the class breadcrumb-item, and active when it is the last one like I showed in the example

Reply:
Okay. Now I got it!